storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -1150,5 +1150,27 @@ function backup_and_restore() {
   verify_db_state $db
 }
 
+### kill all running queries
+function kill_all_queries()
+{
+  run_cmd $MYSQL $MYSQL_ARGS --force --batch  test <<EOF
+  select concat('KILL ',id,';') from information_schema.processlist
+  where user='root' and time > 1 into outfile '$MYSQLD_TMPDIR/killall.sql';
+  source $MYSQLD_TMPDIR/killall.sql;
+EOF
+  rm -f $MYSQLD_TMPDIR/killall.sql
```

Comment:
isn't it better if we just kill the pid of the MySQL client process? I think such function already exists?

Reply:
killing pid is not help full. the process still exists can be seen with show processlist.

Reply:
well, we have to use wait $pid like it is done in other tests.
It looks like this code is duplicated from kill_long_selects.sh. Please move to common place

It is not clear to me if "bg_kill_ok" is used, why do we do "kill_all_queries". Please check with marce

Reply:
these is code from 2013. do you have any idea @altmannmarcelo   but I have removed duplicated code from kill_long_selects.sh.  let me know if it is fine?

Reply:
@rahulmalik87 please test. kill -9 should do imho